Source/LibationCli/ConsoleProgressBar.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.IO;
+
+namespace LibationCli;
+
+internal class ConsoleProgressBar
```

Comment:
CLI UI work is simultaneously frustrating and seductive. I remember [doing one years ago](https://github.com/rmcrackan/Dinah.Core/tree/master/Dinah.Core/ConsoleLib). No idea if it actually holds up lol

Reply:
Huh, didn't even know that was there. I wasn't trying to reinvent the wheel, but it was fun :)

Reply:
Exactly. There was no point in me re-inventing the wheel either -- there are a ton of these -- except that it was fun